pyproject.toml

```diff
@@ -13,6 +13,8 @@ websockets = "^11.0.3"
 aiohttp = "^3.8.4"
 requests = "^2.27.1"
 jupyter-kernel-gateway = "^2.5.2"
+pydantic = "^2.0.3"
+pydantic-settings = "^2.0.2"
```

Comment:
Does this work with langchain? Last time I tried updating to V2 I saw some issues in the codeinterpreter-api because it uses langchain which depends on V1

Reply:
<img width="822" alt="Screenshot 2023-07-21 at 6 52 43 PM" src="https://github.com/shroominic/codebox-api/assets/58709763/a29ca552-c7f9-49fd-b137-be094d6b1804">

Sadly doesn't look like, will fix